src/components/va-alert/va-alert.css

```diff
@@ -26,15 +53,11 @@ div.alert {
   box-sizing: border-box;
 }
 
-div.alert:not(.full-width):not(.bg-only) {
+div.alert:not(.bg-only) {
+  border-color: inherit;
```

Comment:
We aren't adding an additional `full-width` class, so the `:not` selector can go. The `border-color` change is to support the new pattern for status colors - defining them at the host.